_posts/2024-10-21-ai-tools-to-power-academic-research--open-mic-summary.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title:  "AI tools to power academic research"
+categories: [Open Mic Session, Open Mic]
+excerpt: "AI tools to help improve academic research."
+---
+
+On Friday 11 October 2024 speaker [Miroslav Blaško](https://kbss.felk.cvut.cz/web/team#miroslav-blaško) held an Open Mic session with the topic \"AI tools to power academic research\". Video and presentation included.
+
+Then include video using following code:
+
+{% include video id="-UriTtvcnmE" provider="youtube" %}
+
+Video is uploaded to youtube by Michal. The id parameter is part of the link behind v=.
```

Comment:
@MichalMed  please edit this part

You sent this video, but I assume it is not final version right? 
https://www.youtube.com/live/4Kdc_hcOghs